scripts/mkimg.sh

```diff
@@ -23,7 +23,7 @@ populate() {
     echo 'Hello World' > dir1/dir2/dir3/file2
     jot $((1 << 16)) 0 | xargs printf '%015x\n' > file3
     ln -sf dir1/dir2/dir3/file2 link1
-    ln -sf "$(yes ./ | head -n200 | tr -d '\n')/file1" long-link
+    ln -sf "$(yes ./ | head -n508 | tr -d '\n')/file1" long-link
```

Comment:
This creates a 1016 byte symlink.  Why not go all the way to 1024 ?

Reply:
508 * strlen("./") + strlen("/file1") = 1016 + 6 = 1022
1024 doesn't work because it's too long, but I could add an additional '/' to make it 1023, which is the maximum.